control/ssm.go

```diff
@@ -0,0 +1,49 @@
+package control
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
+)
+
+// SSMOption is a function to set ssm options
+type SSMOption func(*SSM)
+
+type SSM struct {
+	Service ssmiface.SSMAPI
+	config  *aws.Config
+}
+
+// New creates an SSM from a list of SSMOption functions
+func NewSSM(opts ...SSMOption) (*SSM, error) {
```

Comment:
No error checking in this function?

Reply:
good call, ill remove the `error` for now